storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+
+.. rst-class:: table table-sm table-hover table-striped
+.. csv-table:: Hitsound Trigger Conditions
+   :header: "Condition Type", "Name of Sampleset", "Additional", "Number of Sampleset", "Triggername"
+   :widths: auto
+
+   "Hitsound", "Soft", "Whistle", "1", "HitsoundSoftWhistle1"
+   "Hitsound", "Drum", "", "", "HitsoundDrum"
+   "Hitsound", "", "Clap", "", "HitsoundClap"
+   "Hitsound", "", "", "4", "Hitsound4"
+   "Hitsound", "Normal", "", "2", "HitsoundNormal2"
+   "Hitsound", "", "Finish", "0", "HitsoundFinish0"
+
+The columns are very easy to understand when you are familiar with hitsounding. Aside from the prefix "Hitsound" you can individually add or leave out a specification of your hitsound to get the exact hitsound or combination of hitsounds you want.
+
+If you don't know what any of these mean, consider reading `a guide on hitsounding <https://osu.ppy.sh/s/224758>`_.
+
+ .. warning:: While the naming of the triggers suggests that it checks which hitsound is played, this is actually not the case!
+    Changing the sampleset of a note via the per-note quickaccess on the top left of the editor(Shift+Q/W/E/R) or via Sample Import will be ignored for the sake of evaluating triggers.
+    You **always** have to change your samplesets via inherited timing sections if you want your Hitsound triggers to work correctly.
+
+.. note:: You might have noticed that there is no option to catch a hitnormal alone. This is very annoying when you want to catch hitnormals along with other notes that actually have finishers.
+  It can be worked around by changing the sampleset for these notes so you can use ``Hitsound5`` or something like that but it requires a lot of manual effort nonetheless.
+
+  If you wonder why this is the case, the answer is that hitnormals work for everyone differently. Currently there is the option to configure the option ``LayeredHitSounds`` in your ``Skin.ini`` to 0 causing hitnormals no longer be played on notes that have additionals.
+  As this is a setting determined by the user in most cases, there is no way to get a consistent experience for all players without the already mentioned workaround anyway.
+
+Example: Hitsounds
+..................
+Aoba wants to play the Taiko drum! She is very inexperienced but maybe you can help her out?
+
+.. image:: img/compound_commands/aobaTaiko.png
+  :scale: 20%
+  :alt: Full of optimism: Aoba
+  :align: left
+
+.. image:: img/compound_commands/drum.png
+  :scale: 30%
+  :alt:  ...and the Taiko drum!
+  :align: right
+
+If you aren't familiar with Taiko mapping, no problem, we got that covered in a few words.
+
+There are 2 types of notes in Taiko, Don (red) and Kat (blue). If it has a whistle and/or clap hitsound attached it is a Kat, otherwise it is a Don note.
+
+
+We can make Aoba hit the drum by catching the respective trigger conditions:
+
+These would be ``HitsoundWhistle`` and ``HitsoundClap`` for Kat.
```

Comment:
Eeeh. I only saw on second sight that this change does not make sense, otherwise I would have commented earlier. 
I see that the connection from the : to the next sentence is iffy but ``HitsoundWhistle`` is not for Don and only pulling Kat into that sentence seems inconsequent.
I did a fair bit of rewording in this part to make it less awkward.